### PR TITLE
TC-557 - get content types available for use

### PR DESCRIPTION
**Pull request by peggles2**
This pull request hooks the landing page up to drupal.

### CODE REVIEW

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 1]
```diff
@@ -12,15 +12,73 @@ import {
...
+
   calloutContent = {
     title: 'What you need to get started:',
     text:
       'This analysis will help you easily prepare an estimate and visual to include in your business plan. We’ll do the math and all you will need is an idea of the following information.'
   }
+
+  async componentDidMount() {
+    const becNodeId = window.nodeId
+    if (becNodeId > 0) {
```
Comment (averysmithproductions) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:32]: I'd suggest to change this to `if (!isEmpty(becNodeId))`.

Be sure to put an import statement for isEmpty from lodash at the top of this file. For example:

`import { isEmpty } from "lodash"`

**Review by averysmithproductions — APPROVED** [review]
LGTM

File: src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx [hunk 2]
```diff
@@ -12,15 +12,73 @@ import {
...
+    }
+  }
+
+  parseBecData(data) {
+    alert(JSON.stringify(data))
+    const becData = {}
+    let i = 0
+
+    //Parse Break-Even Point Paragraph
+    while (data.paragraphs[i].type !== 'sectionHeader') {
```
Comment (averysmithproductions) [inline src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:47]: you could potentially replace your `while` loop with a `for ` loop instead but if that doesn't make sense based on what you're looking for then I think the while is sufficient.

Comment (averysmithproductions) [reply src/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.jsx:47]: for (let i = 0; i < data.paragraphs.length; i++) {
 const paragraph = data.paragraphs[i]
 if (paragraph.type === "sectionHeader") {
  break
 }
 becData.mainParagraphs[i] = data.paragraphs[i].text
}

File: test/jest/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.test.jsx [hunk 3]
```diff
@@ -1,11 +1,28 @@
...
+    global.nodeId = '123'
+
+    fetchRestContentStub = jest.spyOn(fetchContentHelper, 'fetchRestContent')
+    fetchRestContentStub.mockReturnValue({})
+
+    const { getByTestId } = render(<BreakEvenCalculatorPage />)
+    await waitForElement(() => getByTestId('breakevencalc'))
+
+    expect(fetchRestContentStub).toBeCalledTimes(1)
+  })
```
Comment (averysmithproductions) [inline test/jest/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.test.jsx:27]: way to go on your test!

**Review by averysmithproductions — APPROVED** [review]
LGTM

File: test/jest/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.test.jsx (hunk 3 above)
Comment (peggles2) [reply test/jest/client/components/pages/break-even-calc-landing-page/break-even-calc-landing-page.test.jsx:27]: Thanks to your help!! :)

**Pull request merged**